openfisca_us/variables/finance/benefit/school_meals.py

```diff
@@ -14,7 +15,9 @@ class school_meal_subsidy(Variable):
 
     def formula(spm_unit, period, parameters):
         # Get state and poverty ratio for SPM unit.
-        state_group = spm_unit("spm_unit_state_group", period)
+        state_group = spm_unit.value_from_first_person(
```

Comment:
@nikhilwoodruff I think this is where tests are breaking. How does one get household-level variables for a SPM unit? I tried adapting some code from openfisca-uk here but it's not working.

Reply:
I filed this a while ago in core: https://github.com/openfisca/openfisca-core/issues/1041 - the solution is in the first snippet, but I think the second snippet would be much better, will see if I should submit a pull request.

Reply:
That approach breaks as it returns `[0]` as `state_group`:
```
state_group = spm_unit.value_from_first_person(
    spm_unit.members.household("state_group", period)
)
```

Reply:
Oh yeah, I remember there's a few places in openfisca-uk where we have to use Series objects and groupbys, because the value_from_first_person doesn't preserve the Enum attributes.

Reply:
Adding `decode_to_str` worked!